src/SloganForm.js

```diff
@@ -1,24 +1,34 @@
-export default function SloganForm(props) {
+import { useState } from 'react/cjs/react.development';
```

Comment:
should just be `import { useState } from 'react';`  Think autocomplete must have been too helpful here.